x10/perpetual/trading_client/user_module.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+
+from x10.perpetual.accounts import AccountModel
+from x10.perpetual.trading_client.base_module import BaseModule
+from x10.utils.http import WrappedApiResponse, send_get_request
+
+
+class UserModule(BaseModule):
```

Comment:
As an alternative this one can be called `ClientModule`. But it could lead to a confusion with the API client (e.g. `PerpetualTradingClient`).